tpdatasrc/tpgamefiles/rules/d20_combat/damage_critter.py

```diff
@@ -32,28 +51,16 @@ def deal_attack_damage(attacker, tgt, d20_data, flags, action_type):
     
     #Create evt_obj_dam
     evt_obj_dam = tpdp.EventObjDamage()
-    evt_obj_dam.attack_packet.action_type = action_type
+    missing_stub("evt_obj_dam.attack_packet.action_type = action_type #this returns an error")
```

Comment:
Eh? Didn't this function work before?

Reply:
It never worked on my side, I thought you might have fixed it, when I saw it unstubbed. But I still get the same error when testing it.